Makefile

```diff
@@ -53,7 +53,7 @@ endif
 fmt: gci addlicense
 	go mod tidy
 	go fmt ./...
-	find . -type f -name '*.go' -a -exec $(GCI) -local github.com/falcosecurity/falcoctl -w {} \;
+	find . -type f -name '*.go' -a -exec $(GCI) write -s standard -s default -s "prefix(github.com/falcosecurity/falcoctl)" {} \;
```

Comment:
I think we should update the gci version, too.

Reply:
yes.